plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -510,6 +516,10 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    bool                                              _disable_subjective_p2p_billing              = true;
    bool                                              _disable_subjective_api_billing              = true;
    fc::time_point                                    _irreversible_block_time;
+   fc::time_point                                    _accepted_block_time;
+   bool                                              _is_savanna_active                           = false;
+   bool                                              _is_producer_active_finalizer                = false;
+   bool                                              _other_active_finalizers                     = false;
```

Comment:
What does this boolean mean?

Reply:
Are there any active finalizers that are not part of the producers of this nodeos. I'll add a comment.